public_html/administrator/components/com_jed/views/review/view.html.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @package    JED
+ *
+ * @copyright  Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Helper\ContentHelper;
+use Joomla\CMS\Language\Text;
+
+/**
+ * View to edit a review.
+ *
+ * @since  4.0.0
+ */
+class JedViewReview extends JViewLegacy
+{
+	protected $form;
+
+	/**
+	 * The item data.
+	 *
+	 * @var   object
```

Comment:
This is a `CMSObject`

Reply:
use Joomla\CMS\Object\CMSObject; gives me a deprecated warning?

Reply:
That is because it is deprecated for J4 and there is no replacement (yet, who knows when) :) However it is the namespaced class to use.